addons/pos_sale/models/sale_order.py

```diff
@@ -34,6 +34,13 @@ def _compute_amount_unpaid(self):
             total_pos_paid = sum(sale_order.order_line.filtered(lambda l: not l.display_type).mapped('pos_order_line_ids.price_subtotal_incl'))
             sale_order.amount_unpaid = sale_order.amount_total - (total_invoice_paid + total_pos_paid)
 
+    @api.depends('invoice_ids.state', 'currency_id', 'amount_total')
+    def _compute_amount_to_invoice(self):
+        super()._compute_amount_to_invoice()
+        for order in self:
+            for line in order.pos_order_line_ids:
+                order.amount_to_invoice -= line.price_unit
```

Comment:
You should probably use `line.price_subtotal_incl` instead of price_unit. Price unit is the price for 1 quantity only.
Also you need to filter the line that you want to deduct from the `amount_to_invoice`. I guess you want to deduct only the downpayment lines

Reply:
Hello @robinengels thank you for your review.

why do we only need to deduct the downpayment lines? shouldn't every payment made from POS be added to the amount invoiced ??

Reply:
No, when you make the payment for a part of the order in the pos the product already paid appears as delivered and are not counted in the amount to invoice. Here the issue seems to be that the down payment made in PoS isn't taken into account when doing the invoice in the sales app. But in the video on the ticket we can see that the down payment appears in the sale order line, so maybe there is something to do there.